src/scripts/achievements/AchievementHandler.ts

```diff
@@ -177,13 +194,21 @@ class AchievementHandler {
         AchievementHandler.addAchievement('A Small Fortune', 'Obtain 10,000 Pokédollars', new MoneyRequirement(10000), 0.15);
         AchievementHandler.addAchievement('Annual Wage', 'Obtain 100,000 Pokédollars', new MoneyRequirement(100000), 0.20);
         AchievementHandler.addAchievement('Pfft, I Don\'t Need a Bike Voucher', 'Obtain 1,000,000 Pokédollars', new MoneyRequirement(1000000), 0.25);
-        AchievementHandler.addAchievement('A billion Poképennies', 'Obtain 10,000,000 Pokédollars', new MoneyRequirement(10000000), 0.5);
+        AchievementHandler.addAchievement('A billion Poképennies', 'Obtain 10,000,000 Pokédollars', new MoneyRequirement(10000000), 0.3);
+        AchievementHandler.addAchievement('Ready to retire', 'Obtain 1,000,000,000 Pokédollars', new MoneyRequirement(1000000000), 0.4);
+        AchievementHandler.addAchievement('I should go to space...', 'Obtain 100,000,000,000 Pokédollars', new MoneyRequirement(100000000000), 0.6);
 
         AchievementHandler.addAchievement('Dungeon Time', 'Obtain 100 Dungeon Tokens', new TokenRequirement(100), 0.05);
         AchievementHandler.addAchievement('Token Collector', 'Obtain 1,000 Dungeon Tokens', new TokenRequirement(1000), 0.10);
         AchievementHandler.addAchievement('Dungeon Grinder', 'Obtain 10,000 Dungeon Tokens', new TokenRequirement(10000), 0.15);
         AchievementHandler.addAchievement('Dungeon Master', 'Obtain 100,000 Dungeon Tokens', new TokenRequirement(100000), 0.20);
         AchievementHandler.addAchievement('Dungeon Legend', 'Obtain 1,000,000 Dungeon Tokens', new TokenRequirement(1000000), 0.25);
+        AchievementHandler.addAchievement('Dungeon Myth', 'Obtain 100,000,000 Dungeon Tokens', new TokenRequirement(100000000), 0.3);
+        AchievementHandler.addAchievement('Dungeon God', 'Obtain 10,000,000,000 Dungeon Tokens', new TokenRequirement(10000000000), 0.6);
+
+        AchievementHandler.addAchievement('Farm Master', 'Obtain 100,000 Farm Points', new FarmPointsRequirement(100000), 0.20);
+        AchievementHandler.addAchievement('Farm Legend', 'Obtain 1,000,000 Farm Points', new FarmPointsRequirement(1000000), 0.30);
+        AchievementHandler.addAchievement('Farm Myth', 'Obtain 20,000,000 Farm Points', new FarmPointsRequirement(20000000), 0.60);
```

Comment:
Why 20 and not 10? I don't mind, but all the other currency achievements above this start with a 1, this one sticks out.

Reply:
10 mil felt too easy and 50 mil felt too hard. So i ended up with this weird number